lib/resources/remote_config.dart

```diff
@@ -1,23 +1,31 @@
 import 'package:firebase_remote_config/firebase_remote_config.dart';
 
 const String _feedbackEnabled = 'feedback_enabled';
+const String _chatEnabled = 'chat_enabled';
 
 class RemoteConfigService {
   RemoteConfigService._();
 
   static RemoteConfig _remoteConfig;
-  static final defaults = <String, dynamic>{_feedbackEnabled: false};
+  static final defaults = <String, dynamic>{
+    _feedbackEnabled: false,
+    _chatEnabled: false
+  };
   static Map<String, dynamic> overrides;
 
   static bool get feedbackEnabled => overrides != null
       ? true
       : _remoteConfig?.getBool(_feedbackEnabled) ?? defaults[_feedbackEnabled];
 
+  static bool get chatEnabled => overrides != null
+      ? true
+      : _remoteConfig?.getBool(_chatEnabled) ?? defaults[_chatEnabled];
+
   static Future<dynamic> initialize() async {
     try {
       _remoteConfig = await RemoteConfig.instance;
       await _remoteConfig.setDefaults(defaults);
-      await _remoteConfig.fetch();
+      await _remoteConfig.fetch(expiration: const Duration(seconds: 0));
```

Comment:
Why is this necessary?

Also please add the following as the first line of this function:
`if (kIsWeb) return; // Remote config is not yet supported on web.`

Reply:
It was for testing, to fetch data from remote config immediately